subgraph.yaml

```diff
@@ -5,30 +5,34 @@ dataSources:
         - file: >-
             ./node_modules/@orcaprotocol/contracts/deployments/rinkeby/MemberToken.json
           name: MemberToken
-      apiVersion: 0.0.5
+      apiVersion: 0.0.6
```

Comment:
❓ I didn't manually change these, so I assume this auto-increment occurs when a build introduces new changes?

Reply:
not sure, seems like kindof a weird auto pattern tho